Client/Frontend/Home/HomepageViewModel.swift

```diff
@@ -108,6 +108,11 @@ class HomepageViewModel: FeatureFlaggable {
 
         self.recentlySavedViewModel = RecentlySavedCellViewModel(
             profile: profile)
+        let historyDataAdaptor = HistoryHighlightsDataAdaptorImplementation(
+            profile: profile,
+            tabManager: tabManager,
+            urlBar: urlBar,
+            historyHighlightsDataAdaptor: historyDataAdaptor)
         self.historyHighlightsViewModel = HistoryHighlightsViewModel(
```

Comment:
Little issue here

![Screen Shot 2022-08-11 at 5 42 16 PM](https://user-images.githubusercontent.com/11338480/184247417-d345d512-29a3-4bf6-958d-14762a0145d6.png)

Reply:
Oops, had a merge conflict and things got jumbled but I never actually committed the fix I put in for it 🙈